src/types.ts

```diff
@@ -49,8 +49,8 @@ export interface Profile {
     description: string;
     mediatypes: string[];
     defaultMediatype: string;
-    labelPredicate?: string;
-    descPredicate?: string;
+    labelPredicate: string[];
+    descPredicate: string[];
```

Comment:
Now that it's an array for these two fields, can we change it to plural form?

Reply:
Yes I agree, plural form makes more sense

Reply:
Resolved in latest commit